tests/TestCase/ORM/TableTest.php

```diff
@@ -64,20 +64,27 @@ class ProtectedEntity extends Entity
 class TableTest extends TestCase
 {
 
-    public $fixtures = [
```

Comment:
This change in unnecessary. The tests updates you have done in `FixtureManagerTest` suffice.